engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-advanced-sync-options.html

```diff
@@ -0,0 +1,54 @@
+  <div class="radio">
+    <label>
+      <input type="radio" ng-model="syncType" value="standard" />
+      <span translate>
+        Optimized Sync
+      </span>
+    </label>
+    <div class="help-block">
+      <span translate>
+        Standard sync, optimized for speed by bypassing any unneeded steps.
+      </span>
+    </div>
+  </div>
+
+  <div class="radio">
+    <label>
+      <input type="radio" ng-model="syncType" value="skipMetadataCheck" />
+      <span translate>
+        Complete Sync
+      </span>
+    </label>
+    <div class="help-block">
+      <span translate>
+        Sync even if the upstream metadata appears to have no change and is only relevant for yum repositories.  This option will take longer than a normal sync.  Choose this option if:
```

Comment:
Recommend slight rewording: 

Sync even if the upstream metadata appears to have no change.  This option is only relevant for yum repositories and will take longer than an optimized sync.  Choose this option if: